src/server.py

```diff
@@ -533,6 +538,34 @@ def buildbot_name(platform, buildtype, jobname, branch):
     # TODO: do we need to do a jobname conversion?  I don't see a need yet
     return "%s %s %s test %s" % (platform_map[platform], branch, buildtype_map[buildtype], jobname)
 
+@app.route("/data/jobnames/")
+@json_response
+def run_jobnames_query():
+    db = create_db_connnection()
+    cursor = db.cursor()
+    query = "select platform, buildtype, testtype from uniquejobs"
+    cursor.execute(query)
+    branch = 'mozilla-inbound'
+    json_jobnames = {'results': []}
+    jobnames = json_jobnames['results']
```

Comment:
Delete this variable and directly use:
json_jobnames['results'].append(data) below.